src/kwaak_tracing.rs

```diff
@@ -104,10 +106,26 @@ fn init_otel() -> TracerProvider {
         .build()
         .expect("failed to create otlp exporter");
 
+    let service_name = if let Ok(service_name) = std::env::var("OTEL_SERVICE_NAME") {
+        service_name
+    } else {
+        let resource_attributes = std::env::var("OTEL_RESOURCE_ATTRIBUTES")
+            .unwrap_or_default()
+            .split(',')
+            .map(|s| s.split_once("=").expect("invalid OTEL_RESOURCE_ATTRIBUTES"))
```

Comment:
So if you do default (empty string), this panics.

Reply:
oops

Reply:
Eh oh no, you're not correct. We first split on `,` to get the pairs. So if the variable is empty, the split won't yield any results so it would never get to the resource attribute splitter.

Reply:
Hmm you actually are correct, that is surprising..

Reply:
I guess it's not really surprising, if you have a list of one element of course it would not include the separator and yield the one result.

Reply:
I have my moments